spring-boot-test-autoconfigure/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Use random port for embedded Mongod.
+spring.data.mongodb.port = 0
```

Comment:
How is this change related? If that value is sensible why isn't it provided automatically by `@DataMongoTest`?